guides.Rmd

```diff
@@ -196,136 +317,260 @@ Note the use of `%o%` to quickly generate the multiplication table, and that the
-\index{Date/times} \index{Data!date/time} \index{Time} \index{Scales!date/time} \indexf{scale\_x\_datetime}
+The `guides()` helper works in a similar way to the `labs()` helper function described in Section \@ref(scale-name). Both take the name of different aesthetics (e.g., color, x, fill) as arguments and allow you to specify your own value. Where `labs()` provides a shorthand way to specify the `name` argument to one or more scales, the `guides()` function allows you to specify `guide` arguments to one or more scales. In the same way that `labs(colour = "a colour scale name")` specifies the name associated with the colour scale, a command such as `guides(colour = guide_coloursteps())` can be used to specify its associated guide:
 
-*   `date_breaks` and `date_minor_breaks()` allows you to position breaks by
-    date units (years, months, weeks, days, hours, minutes, and seconds). 
-    For example, `date_breaks = "2 weeks"` will place a major 
-    tick mark every two weeks.
+`r columns(3)`
+```{r}
+base <- ggplot(mpg, aes(displ, hwy, colour = cyl)) + geom_point()
 
-*   `date_labels` controls the display of the labels
-    using the same formatting strings as in `strptime()` and `format()`:
+base 
+base + scale_colour_continuous(guide = guide_coloursteps())
+base + guides(colour = guide_coloursteps())
+```
 
-    | String        | Meaning
-    |---------------|-----------------------------------------
-    | `%S`          | second (00-59)
-    | `%M`          | minute (00-59)
-    | `%l`          | hour, in 12-hour clock (1-12)
-    | `%I`          | hour, in 12-hour clock (01-12)
-    | `%p`          | am/pm
-    | `%H`          | hour, in 24-hour clock (00-23)
-    | `%a`          | day of week, abbreviated (Mon-Sun)
-    | `%A`          | day of week, full (Monday-Sunday)
-    | `%e`          | day of month (1-31)
-    | `%d`          | day of month (01-31)
-    | `%m`          | month, numeric (01-12)
-    | `%b`          | month, abbreviated (Jan-Dec)
-    | `%B`          | month, full (January-December)
-    | `%y`          | year, without century (00-99)
-    | `%Y`          | year, with century (0000-9999)
+However, scale guides are more complex than scale names: where the `name` argument (and hence the `labs()` helper function) takes text as input, the `guide` argument (and `guides()` helper) expect you to create a guide object using a **guide function** such as `guide_colourbar()` and `guide_legend()`. These functions offer the user additional control over the fine details of the legend. Legend guides can be used for any aesthetic (discrete or continuous) while the colour bar guide can only be used with continuous colour scales. 
```

Comment:
Should we have a table that shows what determine which guide you get by default?